src/scaffoldmaker/meshtypes/meshtype_3d_cecum1.py

```diff
@@ -0,0 +1,1059 @@
+        'Pig 1': ScaffoldPackage(MeshType_3d_ostium1, {
+            'scaffoldSettings': {
+                'Number of vessels': 1,
+                'Number of elements across common': 2,
+                'Number of elements around ostium': 8,
+                'Number of elements along': 2,
+                'Number of elements through wall': 1,  # not implemented for > 1
+                'Unit scale': 1.0,
+                'Outlet': False,
+                'Ostium diameter': 20.0,
+                'Ostium length': 10.0,
+                'Ostium wall thickness': 2.0,
+                'Ostium inter-vessel distance': 0.0,
+                'Ostium inter-vessel height': 0.0,
+                'Use linear through ostium wall': True,
+                'Vessel end length factor': 1.0,
+                'Vessel inner diameter': 10.0,
+                'Vessel wall thickness': 2.0,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                'Use cross derivatives': False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+            },
+        })
+    }
+
+    @staticmethod
+    def getName():
+        return '3D Cecum 1'
+
+    @staticmethod
+    def getParameterSetNames():
+        return [
+            'Default',
+            'Pig 1']
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName='Default'):
+        centralPathOption = cls.centralPathDefaultScaffoldPackages['Pig 1']
+        ostiumOption = cls.ostiumDefaultScaffoldPackages['Pig 1']
+
+        options = {
+            'Central path': copy.deepcopy(centralPathOption),
+            'Number of segments': 5,
+            'Number of elements around tenia coli': 2,
+            'Number of elements around haustrum': 8,
+            'Number of elements along segment': 8,
+            'Number of elements through wall': 1,
+            'Start inner radius': 35.0,
+            'Start inner radius derivative': 3.0,
+            'End inner radius': 38.0,
+            'End inner radius derivative': 3.0,
+            'Corner inner radius factor': 0.5,
+            'Haustrum inner radius factor': 0.25,
+            'Segment length end derivative factor': 1.0,
+            'Segment length mid derivative factor': 4.0,
+            'Number of tenia coli': 3,
+            'Start tenia coli width': 5.0,
+            'Start tenia coli width derivative': 0.0,
+            'End tenia coli width': 5.0,
+            'End tenia coli width derivative': 0.0,
+            'Tenia coli thickness': 0.5,
+            'Wall thickness': 2.0,
+            'Ileocecal junction': copy.deepcopy(ostiumOption),
+            'Ileocecal junction angular position degrees': 60,
```

Comment:
What happens if it's put over the tenia?

Reply:
Since it does not happen in rea life, I have an assert statement which prevents that from happening:

assert angleAroundInSector > angleToTCEdge + angleOstium * 0.5 and \
               angleAroundInSector < (2*math.pi/tcCount) - angleToTCEdge - angleOstium*0.5,\
               'Ileocecal junction cannot sit on tenia coli'

Reply:
Make sure all asserts like this (I see there are others) are avoided in proper use by checking and limiting the value in checkOptions; asserts are exceptions which shouldn't happen by something a user enters. It's fine to have them in generateBaseMesh to catch callers who misuse the library by setting them wrong not calling checkOptions to limit them.

Reply:
I couldn't limit the angular position of the ileocecal junction in checkOptions as I need to make the cecum segments first before I can use it to calculate the arclength of tenia coli, that's why I put it as an assert when the junction is found sitting over the tenia coli.  

It's the same for the other assert which identifies insufficient elements around the ostium to create a 2-radial elements annulus. I would only know that there is insufficient elements to make that annulus after I have created a tracksurface and know where to place the ostium.